sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -132,6 +132,7 @@ select
   interval '99 11:22:33.123456789' day to second + dateval
 from interval_arithmetic;
 
+-- datetimes(in string representation) + intervals
```

Comment:
shall we move this comment to the next query?

Reply:
oops... fixed.